### PR TITLE
Fix tests in 2.1 branch

### PR DESCRIPTION
**Pull request by serieznyi**
Changes for issue https://github.com/ongr-io/FilterManagerBundle/issues/248

### CODE REVIEW

File: Tests/Unit/Controller/ManagerControllerTest.php [hunk 1]
```diff
@@ -26,35 +33,39 @@ public function testGetResponseAction()
...
+            ->will($this->returnValue($searchResponseMock))
+        ;
 
-        $templating = $this->createMock('Symfony\Bundle\FrameworkBundle\Templating\EngineInterface');
-        $templating
+        $templatingMock = $this->createMock(EngineInterface::class);
+        $templatingMock
             ->expects($this->once())
-            ->method('renderResponse')
+            ->method('render')
```
Comment (serieznyi) [inline Tests/Unit/Controller/ManagerControllerTest.php]: @saimaz in this place I fix test. Other changes - refactoring

**Comment by serieznyi** [comment]
@saimaz why don`t you use squash merge?
I make many commits(some of them are "dirty") in my issues and if you use squash, then you have only one feature commit with my changes.

**Comment by saimaz** [comment]
Usually, I always use squash, this time it was kinda accidentally wrong select.

**Comment by saimaz** [comment]
This is only just history, don't worry about it ;)

**Comment by serieznyi** [comment]
I know it, but I like beautiful history :)

**Pull request merged**